### ISSUE TITLE
Error in running nrbc

### ISSUE THREAD

**Issue by nahidrahmati** [yixiangD/coagulation_cascade#2]
when I am trying this command : 
../../lmp_stam < chan.equil.run

 I saw an error : 
bash: ../../lmp_stam: No such file or directory

When I add this  folder manually in the defined address, it cannot run chan.equil.run as well. Does it need any prerequisite prior to running the case?

**Comment by yixiangD** [comment]
Did the compilation go through? Was there lmp_stam in folder src/? Did you change directory to case/nrbc?

**Comment by nahidrahmati** [comment]
Yes, I did them correctly and I have lmp_stam in src. I think, I have to change the command to : ../../src/lmp_stam < chan.equil.run

In this situation I got this error: 
ERROR: Incorrect args in bond_coeff command

**Comment by yixiangD** [comment]
Okay, I will close the other issue since the compilation works. I will check this error and let you know. Thanks!

**Comment by yixiangD** [comment]
Hi there, could you update the `nrbc_platelet.data` to the newest on the github and run again? It should work now!
